### PR TITLE
Updates for antivirus server

### PR DESCRIPTION
**Pull request by ebrett**
### Context
https://dfedigital.atlassian.net/browse/HFEYP-233

### Changes proposed in this pull request
* use the production server by default
* use non-rest only for testing purposes
* add instructions to README
* remove from docker-compose (greatly increases time)

### CODE REVIEW

File: config/initializers/ratonvirus.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 Ratonvirus.configure do |config|
-  if Rails.env.production?
-    config.scanner = :resty
-    config.storage = :active_storage
-  else
+  if Rails.env.test?
```
Comment (dmallett1) [inline config/initializers/ratonvirus.rb:2]: Discussed last week re Resty being available to test in Test env, as testing 'eicar' is different code.  This does not appear to have changed.

Comment (ebrett) [reply config/initializers/ratonvirus.rb:2]: this is the test environment for automated tests run in rails, nothing to do with the completely separate space called 'test'

Comment (dmallett1) [reply config/initializers/ratonvirus.rb:2]: My mistake no name change required.  So we can now run this is test space, which is great.

File: README.md [hunk 2]
```diff
@@ -236,3 +236,5 @@ https://github.com/niilo/clamav-rest
 
 This will run a clamav-rest server at http://localhost:9000
 
+Set RESTY_SERVICE_URL to "http://localhost:9000/scan"
+
```
Comment (guy-roberts) [inline README.md:240]: Maybe mention that these new env vars need to be set in production ?   

RESTY_USERNAME and RESTY_PASSWORD and PROXY

**Review by guy-roberts — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (ebrett) [reply README.md:240]: At the moment they don't, but will add a note

File: config/initializers/ratonvirus.rb (hunk 1 above)
Comment (ebrett) [reply config/initializers/ratonvirus.rb:2]: The server is currently open to the public, by design

The server is only being used to receive and file and send backs a message on whether that particular file is infected or not.  It does not change any of the existing mechanisms, so files remain in their respective spaces

**Review by guy-roberts — APPROVED** [review]

**Pull request merged**